.github/workflows/update_schedule.yml

```diff
@@ -0,0 +1,21 @@
+name:
+  Update schedule
+
+  # Controls when the workflow will run
+on:
+  # Triggers the workflow every 3 months (starting from october 2022)
+  schedule:
+    - cron: "0 0 10 */3 *" # every 3 months on the 10th of the month at 00:00
```

Comment:
During cohort, the schedule should be ideally updated every week

Reply:
Okay.... 
Then It would be better to
- create a different cron job to update the schedule every week
- another one to update the cohort

What do you think about this?

Reply:
I am not sure to understand the difference between the 2 CRON job. There is only 1 cohort running at a time. We do not need to update schedules of previous cohorts

Reply:
Thanks for the feedback! 

So the cron job Will be running every 7days ? that is once in a week ?

Reply:
Thanks for the feedback! 

So the cron job Will be running every  7days ? that is once in a week ?